CODEOWNERS

```diff
@@ -3,7 +3,8 @@
 /ghcide/session-loader @pepeiborra @fendor
 /hls-graph @pepeiborra
 /hls-plugin-api @berberman
-/hls-test-utils
+/hls-test-utils @fendor
+/test @fendor
```

Comment:
I noticed that `hls-test-utils` has no codeowner, I am volunteering to take over the ownership.
Additionally, added myself as codeowner for `func-test`. Mostly, to make sure my somewhat arbitrary guidance in `test/README.md` cannot be further violated without me noticing.